docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -49,15 +53,100 @@ gerelateerde entiteiten:
 
 ### Wat gebeurt er als een zaak-dossier *blijvend bewaard* wordt?
 
-Het zaak-dossier is overgedragen naar een bewaarplaats.
+Het zaak-dossier wordt overgedragen naar een bewaarplaats.
 
 ### Wat gebeurt er als een zaak-dossier *vernietigd* wordt?
 
 Het vernietigen is het definitief verwijderen van data volgens de NEN2082 (Eis 80):
 
 > Vernietigen van archiefstukken/archiefbestanddelen moet zo gebeuren dat deze niet meer op enigerlei wijze kunnen worden gereproduceerd.
 
-## Gerelateerde attributen
+### Bepalen van de aan archivering gerelateerde attributen voor `ResultaatType`
+
+Zie: [Selectielijst gemeenten en intergemeentelijke organen 2017](https://vng.nl/files/vng/20170706-selectielijst-gemeenten-intergemeentelijke-organen-2017.pdf)
+
+Het ZTC dient ingericht te worden volgens de selectielijst. Het ZTC dient validaties uit te voeren om te zorgen dat inrichting correct is.
+
+*Voorbeeld*
+
+Als `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` wordt ingesteld op `eigenschap` moet het ZTC valideren dat het betreffende `ZaakType` een `Eigenschap` heeft met als `naam`, de waarde die staat in `ResultaatType.BrondatumArchiefprocedure.datumkenmerk`.
```

Comment:
1) `ResultaatType.BrondatumArchiefprocedure.**D**atumkenmerk' den ik.
2) Ik zou zeggen gerelateerde 'Zaaktype' i.p.v. betreffende.
3) Aangezien 'datumkenmerk' behoort bij een specifiek 'Objecttype', moet volgens mij ook worden gevalideerd dat ResultaatType.BrondatumArchiefprocedure.Objecttype` niet leeg is.

Reply:
Gaat om het zaaktype van de onderhavige zaak, vandaar betreffende.

Reply:
Duidelijk, echter volgens mij betreft een `ResultaatType` geen `ZaakType`, maar is die gerelateerd, of hoort-ie erbij. Maar goed, slechts een detail.